deployments/infrastructure.nix

```diff
@@ -5,8 +5,9 @@ let
   iohk-pkgs = import ../default.nix {};
   mkHydraBuildSlave = { config, pkgs, ... }: {
     imports = [
-      ./../modules/hydra-slave.nix
+      ./../modules/amazon-base.nix
```

Comment:
This is wrong in general, previously we had `*-aws-*.nix` that added aws backend specific configuration, so one could use in theory a different backend. Such changes will make it harder to test things on virtualbox or us moving to a provider like GCE.

Reply:
I wonder what this commit really fixes? It should be reverted.

Reply:
@domenkozar, I think we previously discussed and agreed on the `-aws-` aspect separation being too much of a _de-facto_ burden for the benefits it currently provides (none).

If we want to reverse that decision, that's okay, I guess -- let's just be explicit about it, since a lot of the current code assumes AWS.

Reply:
This commit moves development-agnostic things into commonly included locations.

Reply:
Correction: deployment-agnostic things.

Reply:
I have never agreed on this, we should keep in mind to keep deployments agnostic to nixops backend. That is why `-aws` files existed - I don't understand why these changes were reverted.

Reply:
Well, my mistake for understanding it like this, then.

This was quite a bit ago -- during a standup, where I've complained that implementing the core/relay split is getting a bit too complicated with the target aspect separation, and that there already are places (not introduced by me) where the target (physical spec) isolation was not getting respected.  You then agreed that the big change we've been implementing soft of conflicted with the physical spec (aka target) separation.

I understood this as license to inline AWS into common parts.

Reply:
I guess the question now is what is our plan for moving forward with the physical spec aspec separation.

Reply:
I've agreed that we don't have time to fix this now, but that doesn't mean we should make the situation worse.